public/app/features/dashboard/components/ShareModal/ShareLink.tsx

```diff
@@ -135,6 +135,22 @@ export class ShareLink extends PureComponent<Props, State> {
                 </a>
               </div>
             )}
+            {panel && !config.rendererAvailable && (
+              <InfoBox>
+                <p>
+                  <>To be able to render an image of the panel, you must install the </>
```

Comment:
I think this should be better
```suggestion
                  <>To render a panel image, you must install the </>
```